demos/src/js/chartAuthoring-demo-overview.js

```diff
@@ -82,7 +82,7 @@ var demo = demo || {};
                 titleLinkText: "Chart Authoring"
             },
             markup: {
-                description: "The Chart Authoring Tool supports the authoring, export and embedding of highly accessible charts, including sonification of the data. The current version supports creating pie charts and transforming the data into sonified form via a pattern where long notes represent 10% of the pie chart and short tones represent 1%. Anticipated future development includes supporting other kinds of charts, providing other sonification strategies for data, and export functionality.",
+                description: "The Chart Authoring Tool supports the authoring, export and embedding of highly accessible charts, including sonification of the data. The current version supports creating pie charts and transforming the data into sonified form via a pattern where long notes represent 10% of the pie chart and short tones represent 1%. Anticipated future development includes supporting other kinds of charts, providing other sonification strategies for data, and export functionality.<br /><br /><span style=\"font-weight: bold\">Browser Support</span><br />In browsers that support the Web Speech API, each data label is announced followed by the corresponding sonified data. Currently, these browsers include the latest Chrome, Safari and Firefox with web speech configuration enabled. In other browsers, only the sonified data is announced without the data label.",
```

Comment:
`Currently, these browsers include the latest Chrome, Safari and Firefox with web speech configuration enabled` --> `Currently, these browsers include the latest Chrome, Safari, and Firefox (with web speech configuration enabled)`

I think we need the oxford comma and/or parentheses to make it clear Chrome & Safari don't require enabling of web speech.